src/BloomBrowserUI/pageChooser/page-chooser.less

```diff
@@ -153,7 +153,7 @@ BODY {
 	background: url(transparent.gif);
 }
 
-div.ui-dialog div.ui-dialog-buttonset button.ui-button{
+#mainContainer div.ui-dialog div.ui-dialog-buttonset button.ui-button{
```

Comment:
Two thoughts:
1) since dialogs are being done in iframes, why is that the stylesheet for this dialog has to be protected against interfering with another dialog? Why is this stylesheet visible to the other dialog? 

2) If (1) could not be fixed, then, you are using this "mainContainer" id to differentiate one kind of dialog from another. So it should have a name to match, right? Like "#pageChooserDialog".